third_party/2and3/mypy_extensions.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Dict, Type, TypeVar
+
+T = TypeVar('T')
+
+
+def TypedDict(typename: str, fields: Dict[str, Type[T]]) -> Type[dict]: ...
```

Comment:
It's not supported in the current implementation so far, but yes it's in the design.

I have a line item pending to discuss some forward compatibility concerns with keyword args (`**kwfields`) before actually implementing it.